roles/fuse-layer/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Install EGA Fuse Layer
+  yum:
+    state: present
```

Comment:
Not ```latest```? In other words: don't you want the playbook to update the package when a new version is available?